gammapy/makers/background/fov.py

```diff
@@ -193,8 +193,10 @@ def run(self, dataset, observation=None):
 
         """
         mask_fit = dataset.mask_fit
-
-        dataset.mask_fit = self.make_exclusion_mask(dataset)
+        if mask_fit:
+            dataset.mask_fit *= self.make_exclusion_mask(dataset)
+        else:
+            dataset.mask_fit = self.make_exclusion_mask(dataset)
```

Comment:
I noticed this behavior is actually not documented. Could you add one or two sentences in the docstring of `FoVBackgroundMaker` that it modifies the `mask_fit` by combining it with the exclusion mask?